src/server/models/Status.js

```diff
@@ -1,15 +1,20 @@
 import bookshelf from '../db';
 
-const TABLE_NAME = 'status';
+const TABLE_NAME = 'status_changes';
 
 class Status extends bookshelf.Model {
```

Comment:
I think we should prefer Status over StatusChange. Coz we mostly use noun for table, model names.